frontend/apps/app/components/SessionDetailPage/SessionDetailPageClient.tsx

```diff
@@ -43,6 +43,12 @@ export const SessionDetailPageClient: FC<Props> = ({
 }) => {
   const designSessionId = designSessionWithTimelineItems.id
 
+  const handleSelectedVersionChange = useCallback((version: Version | null) => {
```

Comment:
The function name is similar to `handleChangeSelectedVersion` on line 68. Couldn't we make it common?